examples/pipeline_wavernn/main.py

```diff
@@ -0,0 +1,411 @@
+import signal
+from collections import defaultdict
+from datetime import datetime
+from time import time
+from typing import List
+
+import torch
+import torchaudio
+from torch import nn as nn
+from torch.optim import Adam
+from torch.utils.data import DataLoader
+from torchaudio.datasets.utils import bg_iterator
+from torchaudio.models._wavernn import _WaveRNN
+
+from datasets import collate_factory, split_process_ljspeech
+from losses import MoLLoss
+from processing import LinearToMel, NormalizeDB
+from utils import MetricLogger, count_parameters, save_checkpoint
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
+        "--workers",
+        default=4,
+        type=int,
+        metavar="N",
+        help="number of data loading workers",
+    )
+    parser.add_argument(
+        "--checkpoint",
+        default="",
+        type=str,
+        metavar="PATH",
+        help="path to latest checkpoint",
+    )
+    parser.add_argument(
+        "--epochs",
+        default=8000,
+        type=int,
+        metavar="N",
+        help="number of total epochs to run",
+    )
+    parser.add_argument(
+        "--start-epoch", default=0, type=int, metavar="N", help="manual epoch number"
+    )
+    parser.add_argument(
+        "--print-freq",
+        default=10,
+        type=int,
+        metavar="N",
+        help="print frequency in epochs",
+    )
+    parser.add_argument(
+        "--batch-size", default=256, type=int, metavar="N", help="mini-batch size"
+    )
+    parser.add_argument(
+        "--learning-rate", default=1e-4, type=float, metavar="LR", help="learning rate",
+    )
+    parser.add_argument("--clip-grad", metavar="NORM", type=float, default=4.0)
+    parser.add_argument(
+        "--mulaw",
+        default=True,
+        action="store_true",
+        help="if used, waveform is mulaw encoded",
+    )
+    parser.add_argument(
+        "--jit", default=False, action="store_true", help="if used, model is jitted"
+    )
+    parser.add_argument(
+        "--upsample-scales",
+        default=[5, 5, 11],
+        type=List[int],
+        help="the list of upsample scales",
+    )
+    parser.add_argument(
+        "--n-bits", default=8, type=int, help="the bits of output waveform",
+    )
+    parser.add_argument(
+        "--sample-rate",
+        default=22050,
+        type=int,
+        help="the rate of audio dimensions (samples per second)",
+    )
+    parser.add_argument(
+        "--hop-length",
+        default=275,
+        type=int,
+        help="the number of samples between the starts of consecutive frames",
+    )
+    parser.add_argument(
+        "--win-length",
+        default=1100,
+        type=int,
+        help="the number of samples between the starts of consecutive frames",
+    )
+    parser.add_argument(
+        "--f-min",
+        default=40.0,
+        type=float,
+        help="the number of samples between the starts of consecutive frames",
+    )
```

Comment:
since `f-min` is used in torchaudio [transforms](https://github.com/pytorch/audio/blob/master/torchaudio/transforms.py#L214). I prefer this name as the variable name in the transforms. Right?